lib/data/editor/editor_history.dart

```diff
@@ -128,6 +131,7 @@ class EditorHistoryItem {
   final Rect? offset;
   final EditorPage? page;
   final DocChange? quillChange;
+  final Map<Stroke, ColorChange>? colorChange;
```

Comment:
It might be good to `assert` that the length of the `colorChange` map is the same as the list of `strokes`